tests/configure/test_config.py

```diff
@@ -55,10 +57,46 @@ def test_function(x):
 
 
 class TestDatabricksConfig(object):
+    databricks_config_from_password = config.DatabricksConfig.construct_from_password('h', 'u', 'p')
```

Comment:
can you use more meaningful strings? e.g. `test-host`, `test-user`, etc.

Reply:
Sure.